invoice.go

```diff
@@ -108,6 +108,6 @@ type InvoiceFee struct {
 
 // InvoiceFInvoicePaymentDetaile is data that contained in `Invoice` at PaymentDetail
```

Comment:
Typo, you can help to fix the typo altogether please 😬

Reply:
thank you 😬